src/pages/graphql/schema/checkout/mutations/create-payment-order.md

```diff
@@ -86,3 +91,7 @@ Attribute |  Data Type | Description
 `id` | String | The unique order ID generated by PayPal
 `mp_order_id` | String | The unique order ID generated in Commerce if Payment Services is enabled after PayPal returns the `id`
 `status` | String | The status of the payment order. See [order status](https://experienceleague.adobe.com/docs/commerce-admin/stores-sales/order-management/orders/order-status.html) for more information
+`amount` | Float | The amount of the payment order
```

Comment:
I don't love using "the" as previously mentioned, but it's a pattern in this page so I'd leave it.